css/css-writing-modes/forms/select-size-scrolling-and-sizing.optional.html

```diff
@@ -0,0 +1,230 @@
+    <option>Option 65</option>
+    <option>Option 66</option>
+    <option>Option 67</option>
+    <option>Option 68</option>
+    <option>Option 69</option>
+    <option>Option 70</option>
+    <option>Option 71</option>
+    <option>Option 72</option>
+    <option>Option 73</option>
+    <option>Option 74</option>
+    <option>Option 75</option>
+    <option>Option 76</option>
+    <option>Option 77</option>
+    <option>Option 78</option>
+    <option>Option 79</option>
+    <option>Option 80</option>
+    <option>Option 81</option>
+    <option>Option 82</option>
+    <option>Option 83</option>
+    <option>Option 84</option>
+    <option>Option 85</option>
+    <option>Option 86</option>
+    <option>Option 87</option>
+    <option>Option 88</option>
+    <option>Option 89</option>
+    <option>Option 90</option>
+    <option>Option 91</option>
+    <option>Option 92</option>
+    <option>Option 93</option>
+    <option>Option 94</option>
+    <option>Option 95</option>
+    <option>Option 96</option>
+    <option>Option 97</option>
+    <option>Option 98</option>
+    <option>Option 99</option>
+    <option>Option 100</option>
+</select>
+<script>
+for (const writingMode of ["horizontal-tb", "vertical-lr", "vertical-rl", "sideways-lr", "sideways-rl"]) {
+    if (!CSS.supports(`writing-mode: ${writingMode}`))
+        continue;
+
+    const scrollBlockAxis = (() => {
+        switch (writingMode) {
+        case "horizontal-tb":
+            return "scrollTop";
+        case "vertical-lr":
+        case "sideways-lr":
+        case "vertical-rl":
+        case "sideways-rl":
+            return "scrollLeft";
+        }
+    })();
+    const scrollInlineAxis = scrollBlockAxis === "scrollTop" ? "scrollLeft" : "scrollTop";
+
+    const isHorizontal = writingMode === "horizontal-tb";
+    const clientBlock = isHorizontal ? "clientHeight" : "clientWidth";
+    const clientInline = isHorizontal ? "clientWidth" : "clientHeight";
+    const scrollBlock = isHorizontal ? "scrollHeight" : "scrollWidth";
+    const scrollInline = isHorizontal ? "scrollWidth" : "scrollHeight";
+
+    const select = document.querySelector("select");
+
+    test(t => {
+        select.scrollTop = select.scrollLeft = 0;
+        select.style.writingMode = writingMode;
+        t.add_cleanup(() => {
+            select.removeAttribute("style");
+            select.scrollTop = select.scrollLeft = 0;
+        });
+
+        assert_true(
+            select[scrollBlock] > select[clientBlock],
+            "select should be scrollable in block axis"
+        );
+        assert_equals(
+            select[scrollInline], select[clientInline],
+            "select should not be scrollable in inline axis"
+        );
+
+        assert_equals(
+            select[scrollBlockAxis], 0,
+            `scrolling is initially at block start for writing-mode: ${writingMode}`
+        );
+
+        const isReversedInlineDirection = writingMode.endsWith("-rl");
```

Comment:
I don't think this is the right name for this variable, since really we're checking whether we have a "left-to-right block flow direction", in terms the spec uses.